### ISSUE TITLE
Completely messed up when connecting ultrawide

### ISSUE THREAD

**Issue by Davincible** [tcorreabr/Parachute#100]
Works great on my laptop screen by itself, but when I connect my ultrawide through a USB C hub its completely messed up. For some reason it looks better on the screenshot than in real life. 
1) In the screenshot it looks normal, but on my smaller screen, the laptop screen, it basically shows from halfway up. So I only see the lower half of that screen.
2) The windows show incorrectly, all my windows are on my big screen, but somehow they show up on the smaller screen
3) background is incorrect.

![image](https://user-images.githubusercontent.com/25869544/109028522-2c115b00-76c2-11eb-91c4-62c21c62f9a2.png)

**Comment by Davincible** [comment]
Turns out kernel 5.10 was giving me a whole lot of issues. Now it works better on a different kernel, but the blur doesn't work while it is turned on in settings:
![image](https://user-images.githubusercontent.com/25869544/109042374-6aae1200-76d0-11eb-9a7d-5e2e7cfe3cbd.png)

**Comment by koma111** [comment]
Not exactly the same, but I think closely related to #94 and #98. Parachute doesn't play well on multimonitor, even if the aspect ratios and resolutions are the same, let alone two vastly different resolutions or dynamically changing the screen alignment (connecting or disconnecting a screen e.g. the ultrawide in question). Typing `kwin_x11 --replace &` into a terminal or KRunner fixes it temporarily, until the screen setup changes again or the user logs out and back in.

**Comment by Davincible** [comment]
Hmm yeah that command did fix it :D. No clue what it does, but is there someway to do this automatically?

**Comment by koma111** [comment]
The command restarts KWin, the window manager and compositor of KDE. That is the part of the system responsible for displaying windows, fancy effects like blur, handling windows being moved around, keeping track of virtual desktops, etc.
I don't know of a nice way to automate it, maybe through DBus?
If you type the command into KRunner (the amazing search of Plasma, invoked by Alt+F2 or Alt+Space (unless you customized your keyboard shortcuts)), and hit Enter, in the future you will only have to type the first 2-3 letters, because KRunner remembers it.

**Comment by Davincible** [comment]
Yeah I ended up having to run it about every time I reconnect my monitor. It's a minor inconvenience but it works, so thanks for the suggestion
